thewalrus/_hermite_multidimensional.py

```diff
@@ -203,10 +203,10 @@ def hermite_multidimensional_numba(R, cutoff, y, C=1, dtype=None):
     if y.shape[0] != n:
         raise ValueError(f"The matrix R and vector y have incompatible dimensions ({R.shape} vs {y.shape})")
     num_indices = len(y)
-    if isinstance(cutoff, int):
-        cutoffs = tuple([cutoff] * num_indices)
-    else:
+    if isinstance(cutoff, (list, tuple)):
         cutoffs = tuple(cutoff)
+    else:
+        cutoffs = tuple([int(cutoff)] * num_indices)
```

Comment:
Perhaps add a comment here about why casting to an int is necessary (since the cutoff should only be either a list or an int based on the docstring).